README.md

```diff
@@ -222,7 +214,43 @@ This command does not touch repositories in which there are uncommitted changes.
 Examples:
 
 ```bash
-mgit update --recursive
+mgit update -- --recursive
+```
+
+### pull
+
+Pulls changes in existing repositories.
```

Comment:
Unclear to me – Does it change branches?

Reply:
Does `git pull` change a branch?

Reply:
I'll add a note that it does not change anything. Just pull.

Reply:
Do I know what `mgit pull` does under the hood? Don't expect that the user of your tool will know what the tool does under the hood. Right now we're used to the fact that some commands change branches, so having clarity here is important for us.